srcalc/src/main/java/gov/va/med/srcalc/domain/model/DiscreteTerm.java

```diff
@@ -60,6 +64,10 @@ void setVariable(final DiscreteVariable variable)
         fVariable = Objects.requireNonNull(variable);
     }
 
+    /**
+     * Returns the index of this term in relation to other terms in the {@link DiscreteVariable}
```

Comment:
"in relation to..." is kind of vague. It's really the index of the associated option in the `DiscreteVariable.getOptions()` list.